src/common/src/block_builder.cc

```diff
@@ -78,6 +82,19 @@ types::Block BlockBuilder::ConvertRecordToBlock(const std::vector<std::string> &
     }
     block.elevation            = std::stod(record[BLOCK_BUILDER_CSV_FIELD_ELEVATION]);
     block.cumulative_elevation = std::stod(record[BLOCK_BUILDER_CSV_FIELD_CUMULATIVE_ELEVATION]);
+    if (record[BLOCK_BUILDER_CSV_FIELD_CONNECTION] != "")
+    {
+        block.switch_connection = std::stoi(record[BLOCK_BUILDER_CSV_FIELD_CONNECTION]);
```

Comment:
This will throw an exception if the conversion cannot be performed (e.g. the string is not a number).  Do you think we should be able to handle exceptions, or just let the program crash?

Reply:
I'd say given that the csv's shouldn't be modified very frequently, it's okay if it requires proper formatting. If you guys think otherwise though we can accommodate that.